src/viur/core/modules/user.py

```diff
@@ -586,60 +584,62 @@ def login(self, token: str | None = None, *args, **kwargs):
         # FIXME: Check if already logged in
         if not conf.user.google_client_id:
             raise errors.PreconditionFailed("Please configure conf.user.google_client_id!")
+
         if not token:
             request = current.request.get()
             request.response.headers["Content-Type"] = "text/html"
             if request.response.headers.get("cross-origin-opener-policy") == "same-origin":
                 # We have to allow popups here
                 request.response.headers["cross-origin-opener-policy"] = "same-origin-allow-popups"
-            # Fixme: Render with Jinja2?
-            with (conf.instance.core_base_path
-                  .joinpath("viur/core/template/vi_user_google_login.html")
-                  .open() as tpl_file):
-                tplStr = tpl_file.read()
-            tplStr = tplStr.replace("{{ clientID }}", conf.user.google_client_id)
-            extendCsp({"script-src": ["sha256-JpzaUIxV/gVOQhKoDLerccwqDDIVsdn1JclA6kRNkLw="],
-                       "style-src": ["sha256-FQpGSicYMVC5jxKGS5sIEzrRjSJmkxKPaetUc7eamqc="]})
-            return tplStr
-        userInfo = id_token.verify_oauth2_token(token, requests.Request(), conf.user.google_client_id)
-        if userInfo['iss'] not in {'accounts.google.com', 'https://accounts.google.com'}:
-            raise ValueError('Wrong issuer.')
-        # Token looks valid :)
-        uid = userInfo['sub']
-        email = userInfo['email']
 
-        # fixme: use self._user_module.baseSkel() for this later
-        addSkel = skeleton.skeletonByKind(self._user_module.addSkel().kindName)  # Ensure that we have the full skeleton
+            tpl_string = conf.instance.core_base_path.joinpath(
+                "viur/core/template/vi_user_google_login.html"
+            ).open().read()
```

Comment:
```suggestion
            file_path = conf.instance.core_base_path.joinpath("viur/core/template/vi_user_google_login.html")
            with open(file_path) as file:
                tpl_string = file.read()
```
Lets use a context manager that close the file for us.